### PR TITLE
Fix some grammar adversities

### PR DESCRIPTION
**Pull request by PedanticHacker**
Went through all the 1176 lines of code to fix some grammar adversities.

### CODE REVIEW

File: chess/pgn.py [hunk 1]
```diff
@@ -1039,7 +1039,7 @@ def read_game(handle, *, Visitor=GameModelCreator):
             elif token in ["1-0", "0-1", "1/2-1/2", "*"] and len(board_stack) == 1:
                 visitor.visit_result(token)
             else:
-                # Replace zeros castling notation.
+                # Replace zeros with correct castling notation.
```
Comment (ddugovic) [inline chess/pgn.py]: Correct is subjective; the PGN specification uses the letter O whereas the FIDE handbook uses the digit 0.

Comment (PedanticHacker) [reply chess/pgn.py]: Fixed that line. Thanks for the suggestion.

**Comment by PedanticHacker** [comment]
This pull request is now ready for prime time. @niklasf, do you agree?

**Comment by niklasf** [comment]
Thanks.

**Pull request closed without merging**